src/test/java/org/isf/admission/test/Tests.java

```diff
@@ -1287,6 +1290,48 @@ private Admission buildNewAdmission() throws Exception {
 				diseaseOut2, diseaseOut3, operation, dischargeType, pregTreatmentType,
 				deliveryType, deliveryResult, true);
 	}
+	
+	@Test
+	public void checkAdmisionWithDiseaseIpdIn() throws Exception  {
+		//Given
+		int id = setupTestAdmission(false, true);
+		Admission admission = admissionBrowserManager.getAdmission(id);
+		String diseaseCode = "20";
+		//when
```

Comment:
```suggestion
		// When
```
To better match other test's comments